src/components/TransactionsList.js

```diff
@@ -28,7 +29,7 @@ const TransactionsList = () => {
           </th>
         </tr>
 
-        {"... your code here..."}
+        {props.transactions.map( (t,i) => <Transaction key={i} transaction={t} /> ) }
```

Comment:
We generally prefer to use something besides index as a key - either `id` or something else unique. This is because index number can change as the array grows/shrinks